### ISSUE TITLE
keyboard shortcut for command 

### ISSUE THREAD

**Issue by adityatandon007** [zeit/hyper#2785]
- [x] Your Hyper.app version is **2.0.0-canary.14**. Please verify your using the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
  - [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

  ---
  - **Any relevant information from devtools?** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_:

  - **Is the issue reproducible in vanilla Hyper.app?**

## Issue

Hey, team! Thanks for making this wonderful tool. I really liked it a lot. :+1: 
The issue is only when I try to open the new tab using `ctrl+shift+T` it opens 2 tabs i.e. one extra tab is opened always. Is there something needed to be added to fix this. I have added my config below to:

 - **Hyper version**: undefined "2.0.0-canary.14"

 - **OS ARCH VERSION:** linux x64 4.13.0-37-generic
 - **Electron:** 1.8.2  **LANG:** en_IN
 - **SHELL:** /bin/bash   **TERM:** undefined

  <details>
    <summary><strong> ~/.hyper.js contents</strong></summary>
      <pre>
        <code>
          {
  updateChannel: 'stable',

    // default font size in pixels for all tabs
    fontSize: 12,

    // font family with optional fallbacks
    fontFamily: 'Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',

    // terminal cursor background color and opacity (hex, rgb, hsl, hsv, hwb or cmyk)
    cursorColor: 'rgba(248,28,229,0.8)',

    // `BEAM` for |, `UNDERLINE` for _, `BLOCK` for █
    cursorShape: 'BEAM',

    // set to true for blinking cursor
    cursorBlink: false,

    // color of the text
    foregroundColor: '#fff',

    // terminal background color
    backgroundColor: '#000',

    // border color (window, tabs)
    borderColor: '#333',

    // custom css to embed in the main window
    css: '',

    // custom css to embed in the terminal window
    termCSS: '',

    // set to `true` (without backticks) if you're using a Linux setup that doesn't show native menus
    // default: `false` on Linux, `true` on Windows (ignored on macOS)
    showHamburgerMenu: true,

    // set to `false` if you want to hide the minimize, maximize and close buttons
    // additionally, set to `'left'` if you want them on the left, like in Ubuntu
    // default: `true` on windows and Linux (ignored on macOS)
    showWindowControls: '',

    // custom padding (css format, i.e.: `top right bottom left`)
    padding: '12px 14px',

    // the full list. if you're going to provide the full color palette,
    // including the 6 x 6 color cubes and the grayscale map, just provide
    // an array here instead of a color map object
    colors: {
      black: '#000000',
      red: '#ff0000',
      green: '#33ff00',
      yellow: '#ffff00',
      blue: '#0066ff',
      magenta: '#cc00ff',
      cyan: '#00ffff',
      white: '#d0d0d0',
      lightBlack: '#808080',
      lightRed: '#ff0000',
      lightGreen: '#33ff00',
      lightYellow: '#ffff00',
      lightBlue: '#0066ff',
      lightMagenta: '#cc00ff',
      lightCyan: '#00ffff',
      lightWhite: '#ffffff'
    },

    // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
    // if left empty, your system's login shell will be used by default
    //
    // Windows
    // - Make sure to use a full path if the binary name doesn't work
    // - Remove `--login` in shellArgs
    //
    // Bash on Windows
    // - Example: `C:\\Windows\\System32\\bash.exe`
    //
    // Powershell on Windows
    // - Example: `C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\powershell.exe`
    shell: '',

    // for setting shell arguments (i.e. for using interactive shellArgs: ['-i'])
    // by default ['--login'] will be used
    shellArgs: ['--login'],

    // for environment variables
    env: {},

    // set to false for no bell
    bell: 'SOUND',

    // if true, selected text will automatically be copied to the clipboard
    copyOnSelect: false

    // if true, on right click selected text will be copied or pasted if no
    // selection is present (true by default on Windows)
    // quickEdit: true

    // URL to custom bell
    // bellSoundURL: 'http://example.com/bell.mp3',

    // for advanced config flags please refer to https://hyper.is/#cfg
  },

  // a list of plugins to fetch and install from npm
  // format: [@org/]project[#version]
  // examples:
  //   `hyperpower`
  //   `@company/project`
  //   `project#1.0.1`
  plugins: [],

  // in development, you can create a directory under
  // `~/.hyper_plugins/local/` and include it here
  // to load it and avoid it being `npm install`ed
  localPlugins: [],

  keymaps: {
    // Example
    // 'window:devtools': 'cmd+alt+o',
  }
}
        </code>
      </pre>
  </details>

**Comment by jaller94** [comment]
Is it possibly related to this ticket? It would open two tabs after Reloading once.

#2652 Reload causes New Tab to spawn multiple tabs

**Comment by chabou** [comment]
Thank you for your link @jaller94. Certainly related 🙏

**Comment by adityatandon007** [comment]
@jaller94 yeah it is related, but in my case it opens two tabs even if I don't reload, i.e., after launching hyper and then pressing `Ctrl+T`.
Should I close this in reference to #2652. Let me know.

**Comment by adityatandon007** [comment]
@chabou has this been resolved, or is there any quick fix to resolve this

**Comment by ntkoopman** [comment]
I have the same problem, but it also happens on other keyboard shortcuts: ctrl-shift-arrow would move two tabs, ctrl-shift-d will open two panes etc.

If you reload again, everything happens three times, etc. So probably an event handler is not removed somewhere when reloading.

**Comment by stereokai** [comment]
Happens to me too, after reloading.

**Comment by bowersjj** [comment]
I've come across this, and its very familiar to the split pane issue I'm having in 2.0.0 (stable) [#2958](https://github.com/zeit/hyper/issues/2958)

**Comment by chabou** [comment]
It will be fixed by #3006

**Comment by darkvertex** [comment]
Also came across this annoying issue, then installed the canary version and it is indeed fixed there.
